packages/pn-personafisica-webapp/src/api/delegations/__test__/Delegations.api.test.ts

```diff
@@ -17,36 +17,35 @@ import {
   REJECT_DELEGATION,
   REVOKE_DELEGATION,
 } from '../delegations.routes';
+import { mockApi } from '../../../__test__/test-utils';
 
 async function getDelegates(response: Array<Delegation> | null) {
-  const axiosMock = new MockAdapter(apiClient);
-  axiosMock.onGet(DELEGATIONS_BY_DELEGATOR()).reply(200, response);
+  mockApi(apiClient, 'GET', DELEGATIONS_BY_DELEGATOR(), 200, undefined, response);
```

Comment:
Here you are not setting mock instance and so it isn't neither restored or resetted

Reply:
changes made into: [430db73](https://github.com/pagopa/pn-frontend/pull/938/commits/430db73dd4e6ee0215846db2b30db4717bdd1763)